resources/netlog_defs.ts

```diff
@@ -35,6 +35,13 @@ export type LogDefinition = {
   blankFields?: readonly number[];
   // This field and any field after will be treated as optional when creating capturing regexes.
   firstOptionalField: number | undefined;
+  // These fields are treated as repeatable fields
+  repeatingFields?: {
+    startingIndex: number;
+    label: string;
+    names: readonly string[];
+    sortFn?: (L: unknown, R: unknown) => number;
```

Comment:
This is technically narrow-able, but since the `L` and `R` parameters can be user-supplied in a user trigger, I felt that defaulting to `unknown` was a better option for safety reasons. If you'd rather handle that check elsewhere, or not enforce the check at all, that's fine.

As for casing, I'll probably just expand to full `left` and `right` instead of `L`, `R`?

Reply:
Yeah, I think that's fine to be safe about input from users (with a comment about why).  Maybe it should throw an exception / print something to the console if it's not a string then? I just think it'd be better handled in the caller of the sort function rather than in every sort function.

Full `left` and `right` sounds fine to me.  `a` and `b` are also common param names to a sort function.